modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -506,24 +511,21 @@ Keystore Alias (?):
 Keystore Alias Password (?):
 INFO: Mule agent was successfully configured to use a TLS channel for REST communications.
 ----
-_Keystore location_
-
-The location of the keystore file to encrypt the communication channel. The keystore must be in JKS format. It is mandatory to provide one.
-
-_Truststore location_
-
-The location where of the truststore file to accept incoming requests from the administration console. The truststore must be in JKS format and must not have a password.
-
-_Keystore Password_
-
-The password to read the keystore. The password is used by the agent to open the keystore.
-
-_Keystore Alias_
-
-The alias of the key stored in the keystore.
-
-_Keystore Alias Password_
 
+* *Keystore location*
++
+Specifies the location of the keystore file to encrypt the communication channel. The keystore file must be in JKS format. (Required)
```

Comment:
> Suggest including "Required" either right next to the bullet text or at the beginning of the definition, rather than at the end.
> Does this imply that the other values are optional? Do we need to do more than imply?

@dhimate - Are any other values Required? Should we identify the Optional ones specifically?